docs/docs-components/data/types.js

```diff
@@ -12,3 +12,64 @@ export type DesignOverview = {|
   path: string,
   svg: SVGElement,
 |};
+
+type StatusType = 'notAvailable' | 'partial' | 'planned' | 'ready';
+
+/**
+ * All components should be in a single list. That list can be filtered by category/platform/etc where needed.
+ */
+
+type ComponentAccessibility = {|
+  a11yComprehension?: StatusType,
```

Comment:
As a general rule, these types don't have nullable _values_, only nullable _fields_. Nullable fields are easy to deal with using optional chaining, and this simplifies downstream logic — if a field exists, its value exists.